.github/workflows/ci.yml

```diff
@@ -5,7 +5,21 @@ on:
   pull_request:
 
 jobs:
+  test-repository:
```

Comment:
`repository` is confusing, since we also have the `RedisRepository`. Maybe `test-project`?

Reply:
I have three components in this repo: `api`, `repository` and `indexer`.

I can rename `repository` to something like `db` or `datastore`, what name do you prefer?

Reply:
Ooooh, sorry I thought this was testing `.` and not `./repository`